tests/test_run_metrics.py

```diff
@@ -0,0 +1,16 @@
+from traccuracy import run_metrics
+from traccuracy.matchers import CTCMatcher
+from traccuracy.metrics import CTCMetrics
+
+from tests.test_utils import get_movie_with_graph
+
+
+def test_run_metrics():
```

Comment:
This test is minimal indeed. That said, what do we actually want to test in run_metrics that is not tested in the actual Metrics/Matchers? The only thing I can really thing of is that it works for multiple metrics, and then maybe validation when we implement it properly (although we will probably test the validation function separately, right?).

Reply:
I think I should add a case where we actually have more than one metric. I'm also thinking about switching to DummyMetric and DummyMatcher so we don't waste any time actually running the algorithms even on a simple test case. 

I had thought about trying to do more validation of the inputs which would add some extra cases for test coverage.

Reply:
DummyMetric and DummyMatcher make sense to me, although having DummyMetric return a non-empty dictionary would be good just for sanity checking the results output format.

We can add tests for validation as we implement it - but it's good to have the space to put it now.